develop/items/first-item.md

```diff
@@ -0,0 +1,142 @@
+
+This will not work if you've marked the item as damageable, as the stack size is always 1 for damageable items to prevent duplication exploits.
+:::
+
+@[code transcludeWith=:::2](@/reference/latest/src/main/java/com/example/docs/item/ModItems.java)
+
+However, when you go in-game, you can see that our item doesn't exist! This is because you don't statically initialize the class.
+
+To do this, you can add a public static initialize method to your class and call it from your `ModInitializer` class. Currently, this method doesn't need anything inside it.
+
+@[code transcludeWith=:::3](@/reference/latest/src/main/java/com/example/docs/item/ModItems.java)
+
+@[code transcludeWith=:::1](@/reference/latest/src/main/java/com/example/docs/item/FabricDocsReferenceItems.java)
+
+Calling a method on a class statically initializes it if it hasn't been previously loaded - this means that all `static` fields are evaluated. This is what this dummy `initialize` method is for.
+
+## Adding the Item to an Item Group
+
+::: info
+If you want to add the item to a custom `ItemGroup`, checkout the [Custom Item Groups](./item-groups.md) page for more information.
+:::
+
+For example purposes, we will add this item to the ingredients `ItemGroup`, you will need to use Fabric API's item group events - specifically `ItemGroupEvents.modifyEntriesEvent`
+
+This can be done in the `initialize` method of your items class.
+
+@[code transcludeWith=:::4](@/reference/latest/src/main/java/com/example/docs/item/ModItems.java)
+
+Loading into the game, you can see that our item has been registered, and is in the Ingredients item group:
+
+![Item in the ingredients group](/assets/develop/items/first_item_0.png)
+
+However, it's missing the following:
+
+- Item Model
+- Texture
+- Translation (name)
+
+## Naming The Item
+
+The item currently doesn't have a translation, so you will need to add one. The translation key has already been provided by Minecraft: `item.mod_id.suspicious_substance`.
+
+Create a new JSON file at: `src/main/resources/assets/<mod id here>/lang/en_us.json` and put in the translation key, and it's value:
+
+```json
+{
+    "item.mod_id.suspicious_substance": "Suspicious Substance"
+}
+```
+
+You can either restart the game or build your mod and press <kbd>F3</kbd> + <kbd>T</kbd> to apply changes.
+
+## Adding a Texture and Model
+
+To give your item a texture and model, simply create a 16x16 texture image for your item and save it in the `assets/<mod id here>/textures/item` folder. Name the texture file the same as the item's identifier, but with a `.png` extension.
+
+For example purposes, you can use this example texture for `suspicious_substance.png`
+
+<DownloadEntry type="Texture" visualURL="/assets/develop/items/first_item_1.png" downloadURL="/assets/develop/items/first_item_1_small.png" />
+
+When restarting/reloading the game - you should see that the item still has no texture, that's because you will need to add a model that uses this texture.
```

Comment:
Maybe explain what the model does a bit? Its all good giving them some json to copy/paste but I do think a single stance to explain the model would go a long way.